Modules/ITS/src/ITSTrackTask.cxx

```diff
@@ -240,6 +245,25 @@ void ITSTrackTask::createAllHistos()
   addObject(hVertexContributors);
   formatAxes(hVertexContributors, "Number of contributors for vertex", "Counts", 1, 1.10);
   hVertexContributors->SetStats(0);
+  
+  hAssociatedClusterFraction = new TH1D("AssociatedClusterFraction", "AssociatedClusterFraction", 100, 0, 1);
+  hAssociatedClusterFraction->SetTitle("The fraction of clusters in tracking event by event");
+  addObject(hAssociatedClusterFraction);
+  formatAxes(hAssociatedClusterFraction, "", "Clusters in track / Clusters", 1, 1.10);
+  hAssociatedClusterFraction->SetStats(0);
+  
+  hNtracks = new TH1D("Ntracks", "Ntracks", mNtracksMAX, 0, mNtracksMAX);
```

Comment:
Cast mNtracksMAX to "int" when using it for defining the number of bins. Not sure if it works like that.